### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by meganle**
To run it locally, build it
`docker build -t eatery .`
and then to run it
`docker run -d -p 5000:5000 eatery`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+EXPOSE 5000 
+
+CMD flask run --host=0.0.0.0
```
Comment (kq-li) [inline Dockerfile]: prefer `python app.py`, think this does some work under the hood that we may not want (env vars) -- though we'll probably do gunicorn soon anyway

**Review by kq-li — APPROVED** [review]
lgtm, maybe `flask run` => `python app.py` for transparency

**Comment by kq-li** [comment]
also you may not want to daemonize the container (-d flag) when running it so you can actually see output and such, probably best to just run it on a separate terminal for local testing purposes

**Comment by young-k** [comment]
LGTM, check out kenneth's comments (insightful as always)

**Pull request merged**